README.md

```diff
@@ -27,7 +27,73 @@ By default, GitHub  does not apply any branch protection policies to newly creat
+1.  Install and launch VS Code (Visual Studio Code) from within the Anaconda Navigator. **Note** after installing VS Code, a Getting Started walkthrough is shown. Click the back button to escape.
+1.  Navigate to the `Git CMD` from the start menu and execute the below commands. Once you have executed the commands close `Git CMD`.
+
+    **Note: You need to change the name/email to match your own and you need to include the quotation marks. You may like to copy the commands into a word document to edit them.**
+
+```shell
+git config --global user.name "Your Name"
+git config --global user.email "Your.Name@levellingup.gov.uk"
+``` 
+
+[Make Chrome your default browser]: https://support.google.com/chrome/answer/95417?hl=en-GB&co=GENIE.Platform%3DDesktop
+
+### Downloading the code from GitHub
+
+1.  Create a folder on your desktop, for storing source code within if you don't have one already.
+1.  From VS Code open the [Explorer window][explorer_window], the overlapping pages icon on left hand menu. Select the option to "Clone Repository". Click "Clone from GitHub"
+1.  If prompted, authorize with GitHub.
+1.  You should be prompted to enter a repository name. Type "communitiesuk/&lt;Your repository name&gt;". Then click on communitiesuk/&lt;Your repository name&gt;.
+1.  As a destination, select your folder for storing the source code. Select "Sign in with browser" if the GitHub authorisation popup is shown.
+1.  This pulls the code from GitHub to your local folder.
+    Click "Open folder" option, and navigate to your newly created folder containing the repository code.
+1.  Select "Yes, I trust the authors".
+
+[explorer_window]: https://code.visualstudio.com/docs/getstarted/userinterface#_explorer
+
+### Installing packages
+
+1.  [Open a command prompt terminal within VS Code][open-terminal], in which you'll start executing some commands. By default, the initial terminal will be a powershell terminal, and you will need to [switch to a command prompt shell][terminal-switch]. 
+1.  Update the name field in environment.yml to the dashboard name
+1.  Create a new conda environment by typing `conda env create -f environment.yml` into the terminal and executing the command by pressing the Enter key.
+1.  Activate your conda environment with `conda activate <dashboard name> `
+1. Close VS Code. Open Anaconda Navigator, select "&lt; Dashboard name &gt;" for the 'Application on' drop down menu, then select "Launch" VS Code. Click the bin icon on the terminal toolbar to close the terminal. Click the plus icon on the terminal toolbar to launch a new terminal.
+1.  Install the [Microsoft Python][python_extension] extension for VS Code.
+1.  Follow the [instructions for configuring the Python interpreter][configure_python_interpreter].
+
+ **Note: If in DLUHC and get an error when creating conda environment check conda version. Type 'conda --version' in VS Code terminal. conda 4.5.12 needs to be upgraded to 4.10.3. Contact DAP-support@communities.gov.uk**
```

Comment:
Do we want to include DAP supports email address if this repo is released to external users?

Reply:
Probably not - thanks for flagging!

Reply:
The note has been removed